templates/blog/newsletter-form.html

```diff
@@ -77,16 +77,5 @@ <h3 class="p-card--title p-heading--four">Select topics you're <br />interested
   </div>
 </div>
 
-<script src="https://assets.ubuntu.com/v1/5d7e5bbf-jquery-2.2.0.min.js"></script>
-<script src="https://assets.ubuntu.com/v1/d55f58bb-jquery.validate.js"></script>
 <script src="https://www.google.com/recaptcha/api.js?onload=CaptchaCallback&render=explicit" defer>
-<script>
-  $("#mktoForm_1212").validate({
```

Comment:
I love the idea of removing this, but why don't we need it any more?

Reply:
I am trying to standardised our forms a little. HTML5 validation validates for different types of inputs automatically. This validation method pre-dates HTML5.

Reply:
Are UX cool with HTML5 validation?

Reply:
Good question, what do you think @clagom. Happy to demo it if you want to see the difference.